python/tk_desktop2/websockets/util.py

```diff
@@ -20,9 +22,8 @@ def create_reply(data, encrypt_fn=None):
     :param encrypt_fn: Optional Encryption method.
     :returns: Server ready payload
     """
-    # ensure_ascii allows unicode strings.
-    payload = json.dumps(data, ensure_ascii=False, default=_json_date_handler).encode(
-        "utf8"
+    payload = six.ensure_str(
+        json.dumps(data, ensure_ascii=False, default=_json_date_handler)
```

Comment:
You should test with unicode chars in your software entity names and descriptors. I think you'll have a bug here. I've had to do this in my exploration branch for `tk-framework-desktopserver`. This also removed the need to encode to utf8 (or `ensure_str`) since `ensure_ascii` converts the output to ASCII7, which is utf8 compliant.
```
payload = json.dumps(
            data,
            ensure_ascii=True,
            default=self._json_date_handler,
        )
```